### ISSUE TITLE
Wrong pfr_id

### ISSUE THREAD

**Issue by SCasanova** [nflverse/nflfastR-roster#23]
Spencer Brown, a UDFA RB, has the incorrect pfr_id "BrowSp00" which corresponds to an  offensive tackle for the Bills

``` r
library(magrittr) 
nflreadr::load_rosters() %>% 
    dplyr::filter(pfr_id == 'BrowSp00') %>% 
    dplyr::select(full_name, team, position, pfr_id)
#> # A tibble: 1 × 4
#>   full_name     team  position pfr_id  
#>   <chr>         <chr> <chr>    <chr>   
#> 1 Spencer Brown CAR   RB       BrowSp00
nflreadr::load_snap_counts() %>% 
     dplyr::filter(pfr_player_id == 'BrowSp00') %>% 
    dplyr::select(player, team, position, pfr_player_id) %>% 
    unique()
#> # A tibble: 1 × 4
#>   player        team  position pfr_player_id
#>   <chr>         <chr> <chr>    <chr>        
#> 1 Spencer Brown BUF   T        BrowSp00
```

<sup>Created on 2021-11-16 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

**Comment by tanho63** [comment]
Corrected the wrong one on the ffverse side. Unsure if that's driving this rosters issue, we'll see after everything rebuilds over the next week

**Comment by mrcaseb** [comment]
Fixed in 3b6f8818e3551adb50b6113ec7fc13337e5c0902

``` r
nflreadr::load_rosters() |> 
  dplyr::filter(full_name == "Spencer Brown") |> 
  dplyr::select(team, position, full_name, gsis_id, pfr_id)
#> # A tibble: 2 x 5
#>   team  position full_name     gsis_id    pfr_id  
#>   <chr> <chr>    <chr>         <chr>      <chr>   
#> 1 BUF   OT       Spencer Brown 00-0036976 BrowSp00
#> 2 CAR   RB       Spencer Brown 00-0036543 BrowSp01
```
